README.md

```diff
@@ -0,0 +1,175 @@
+<a name="readme-top"></a>

```

Comment:
- [ ] The documentation does not have any examples:

![image](https://github.com/mercymugambi/backend_house_reservations_rails_app/assets/57421823/7100ae37-11f5-4308-94ed-3c3a504c68b8)

Kindly, add content to your swagger documentation.

That way, we will meet this particular requirement. 👍